dtfs-central-api/src/v1/validation/utilisation-report-service/utilisation-report-validator.js

```diff
@@ -0,0 +1,94 @@
+const { UTILISATION_REPORT_HEADERS } = require('../../../constants/utilisationReportHeaders');
+const {
+  validateUkefId,
+  validateExporter,
+  validateBaseCurrency,
+  validateFacilityUtilisation,
+  validateTotalFeesAccrued,
+  validateMonthlyFeesPaid,
+  validatePaymentCurrency,
+  validateExchangeRate,
+} = require('./utilisation-data-validator');
+
+/**
+ * Validates the month of the utilisation report. Returns null if valid, otherwise returns an error message.
+ * @param {Integer} month - Month of utilisation report.
+ * @returns {String | null} - Error message or null if valid.
+ */
+const validateMonth = (month) => {
+  if (!month) {
+    return 'Month is required';
+  }
+  if (month < 1 || month > 12) {
```

Comment:
Could add some safety here to reject non-numbers, or numbers with decimal place. e.g.
```
if (!Number.isInteger(month) || month < 1 || month > 12) {
```
Same comment for the year below.